packages/block-library/src/blocks/slideshow/edit.js

```diff
@@ -88,8 +74,6 @@ class Edit extends Component {
 				<SlideshowPreview
 					{ ...newProps }
 					previewImage={ galleryImages[ selectedIndex ] }
```

Comment:
selectedIndex seems to still be relevant for editing the focal point for each image, you could try to refactor this component into a functional component and use `useState()` for cleaner code

Reply:
setIndex function passed to the GalleryPreview component inside InspectorControls should also trigger slick to go to that specific slide